File: latihan3 b/Latihan3 PHP/RAM.php
<?php

class RAM
{ 
	/*kelas yang digunakan untuk mengimplementasikan RAM/
	/*atribut RAM*/
		
		private $capacity = "";		//capacity
		private $Rprice = 0;		//Rprice
	
		public function __construct($capacity ="", $Rprice = 0){
		/*costructor*/
		
			$this->capacity = $capacity;
			$this->Rprice = $Rprice;
		}
	
	/*mengeset nilai capacity*/
	public function setcapacity($capacity)
	{
		$this->capacity = $capacity;
	}
	
	/*mengeset nilai Rprice*/
	public function setRprice($Rprice)
	{
		$this->Rprice = $Rprice;
	}
	
	/*mengembalikan nilai capacity*/
	public function getcapacity()
	{
		return $this->capacity;
	}
	
	/*mengembalikan nilai Rprice*/
	public function getRprice()
	{
		return $this->Rprice;
	}
	
	function __destruct(){
		/*desstructor*/
		}
}
?>

File: latihan3 b/Latihan3 Py/RAM.py
class RAM():
#kelas yang digunakan untuk mengimplementasikan RAM
#atribut RAM
	

	Rcapacity = 0	#capacity
	Rprice = 0		#price
	
	def __init__(self, Rcapacity=0, Rprice = 0):
	#costructor
		self.Rcapacity = Rcapacity
		self.Rprice = Rprice
        
    #mengeset nilai capacity
	def setcapacity(self, Rcapacity):
		self.Rcapacity = Rcapacity
	
	#mengeset nilai price
	def setprice(self, Rprice):
		self.Rprice = Rprice
        
    #mengembalikan nilai capacity
	def getcapacity(self):
		return self.Rcapacity   

	#mengembalikan nilai price
	def getprice(self):
		return self.Rprice

	
	